packages/cxl-lumo-styles/scss/themes/vaadin-accordion-panel.scss

```diff
@@ -83,3 +83,36 @@
     border-bottom-color: var(--lumo-primary-color-10pct);
   }
 }
+
+/**
+ * Theme "cxl-faq".
+ */
+:host([theme~="cxl-faq"]) {
+  border-bottom: 0;
+
+  [part="toggle"] {
+    margin-right: var(--lumo-space-s);
```

Comment:
Hm, previous themes seem to be using `--lumo-space-m`. Is `s` intentional here?

Reply:
After building our default styles, I now agree that design screen toggle architecture is more impactful. Let's bring back the extra style rules here, `border-radius: unset; background-color: var(--lumo-shade);` etc

Reply:
> Hm, previous themes seem to be using `--lumo-space-m`. Is `s` intentional here?

I remeber us talking about it and you said that don't put too much white space. With s, the question and answer section is vertically aligned aligned.

>  Let's bring back the extra style rules here
if we bring back black boxes, we have to reconsider s or m anyway